File: backends/go-gin/internal/usecase/invitation/dto.go
package invitation

// CurrentDto は現在の招待情報取得のユースケース入力です。
type CurrentDto struct {
	Role int
}

// IssueDto は招待発行のユースケース入力です。
type IssueDto struct {
	Role int
}

// FindByTokenDto は招待トークン検索のユースケース入力です。
type FindByTokenDto struct {
	Token string
}

File: backends/rust-axum/src/usecase/invitation/dto.rs
//! 招待ユースケース DTO モジュール。
//!
//! # Author
//! Satoshi Nagashiba <satoshi.nagashiba@gmail.com>

/// 招待トークン検索入力 DTO。
pub struct FindByTokenDto {
    pub token: String,
}

/// 招待 current/issue 入力 DTO。
pub struct RoleDto {
    pub role: u8,
}
